src/arrayMethodJoin.js

```diff
@@ -4,8 +4,22 @@
  * Implement method join
  */
 function applyCustomJoin() {
-  [].__proto__.join2 = function(separator) {
-    // write code here
+  [].__proto__.join2 = function(separator = ',') {
+    let result = '';
+
+    for (let i = 0; i < this.length; i++) {
+      if (this[i] === null || this[i] === undefined) {
+        result += '';
+      } else {
+        result += this[i];
+      }
```

Comment:
```suggestion
      if (this[i] !== null || this[i] !== undefined) {
        result += this[i];
      }
```
You can use only 1 check instead of 2. If you will check that your element is NOT null or NOT undefined then you add this element to the string. If element is null/undefined - you don't add nothing